core/controlplane/config/templates/cloud-config-controller

```diff
@@ -898,6 +898,7 @@ write_files:
           command:
           - /hyperkube
           - apiserver
+          - --apiserver-count={{if .MinControllerCount}}{{ .MinControllerCount }}{{else}}{{ .ControllerCount }}{{end}}
```

Comment:
@mumoshu I believe this may replicate the same behaviour of [`MinControllerCount`](https://github.com/coreos/kube-aws/blob/master/core/controlplane/config/config.go#L568) method? i.e. `.MinControllerCount` is never nil.